src/components/cardChecklist/cardCheckList.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom";
+
+import * as Scry from "scryfall-sdk";
+import s from "./cardCheckList.module.scss";
+
+interface Props {
+  card: Scry.Card;
+}
+
+export const CardCheckList = (props: Props) => {
+  const card: Scry.Card = props.card;
+  const img_uri: string | undefined = props.card.image_uris?.png;
```

Comment:
```suggestion
  const img_uri  = props.card.image_uris?.png;
```
Если png в типе Scry.Card объявлено как необязательное, то у тебя img_uri возьмет его тип как string | undefined итак. Не нужно дублировать типы.